main.js

```diff
@@ -84,19 +85,59 @@ define(function (require, exports, module) {
             }
             node = node.parentElement;
         }
-        
+
         // Close settings dropdown, if open
         _hideSettings();
     }
-    
+
+    function _handleGFMCheckboxClick(e) {
```

Comment:
I'd prefer to have the task lists be read-only. This extension is for _previewing_ markdown files, and does not have any edit functionality. This is intentional. Making task list items clickable starts down a slippery slope of supporting other editing features.

Github only supports editable task lists for issues, pull requests, and comments. Task lists in all other markdown documents are [read-only](https://github.com/blog/1825-task-lists-in-all-markdown-documents).